src/olympia/lib/settings_base.py

```diff
@@ -1638,30 +1640,20 @@ def get_redis_settings(uri):
     #'api_key:secret': os.path.join(ROOT, 'path', 'to', 'file.key'),
 }
 
-# Time in seconds for how long a JWT auth token can live.
-# When developers are creating auth tokens they cannot set the expiration any
-# longer than this.
-MAX_JWT_AUTH_TOKEN_LIFETIME = 60
-
+# Time in seconds for how long a JWT auth token created by developers with
+# their API key can live. When developers are creating auth tokens they cannot
+# set the expiration any longer than this.
+MAX_APIKEY_JWT_AUTH_TOKEN_LIFETIME = 60
 
 # django-rest-framework-jwt settings:
 JWT_AUTH = {
-    # We don't want to refresh tokens right now. Refreshing a token is when
-    # you accept a request with a valid token and return a new one with a
-    # longer expiration.
-    'JWT_ALLOW_REFRESH': False,
-
-    # JWTs are only valid for one minute.
-    'JWT_EXPIRATION_DELTA': datetime.timedelta(
-        seconds=MAX_JWT_AUTH_TOKEN_LIFETIME),
-
-    'JWT_ENCODE_HANDLER': 'olympia.api.jwt_auth.handlers.jwt_encode_handler',
-    'JWT_DECODE_HANDLER': 'olympia.api.jwt_auth.handlers.jwt_decode_handler',
-    'JWT_PAYLOAD_HANDLER': 'olympia.api.jwt_auth.handlers.jwt_payload_handler',
-    'JWT_ALGORITHM': 'HS256',
```

Comment:
I liked setting this explicitly because there have been some JWT exploits around default algorithms in the past.